flow/project.py

```diff
@@ -829,11 +986,12 @@ def _generate_id(self, job, operation_name=None, index=0):
         if max_len < len(job_op_id):
             raise ValueError("Value for MAX_LEN_ID is too small ({}).".format(self.MAX_LEN_ID))
 
+        aggregate_hash = self.get_aggregate_hash(jobs)
```

Comment:
There is really not a need for this function. We use the concatenated string in `full_name` for a unique hash.